vnext/ReactUWP/Views/TextViewManager.cpp

```diff
@@ -22,11 +22,37 @@ namespace uwp {
 class TextShadowNode : public ShadowNodeBase {
   using Super = ShadowNodeBase;
 
- public:
-  TextShadowNode() = default;
+ private:
+  ShadowNode *m_firstChildNode;
+
+ public:  
+  TextShadowNode() {
+    m_firstChildNode = nullptr;
+  };
   bool ImplementsPadding() override {
     return true;
   }
+
+  
+  void AddView(ShadowNode &child, int64_t index) override {
+    if (index == 0) {
+      auto run =
+          static_cast<ShadowNodeBase &>(child).GetView().try_as<winrt::Run>();
+      if (run != nullptr) {
+        m_firstChildNode = &child;
+        auto textBlock = this->GetView().as<winrt::TextBlock>();
+        textBlock.Text(run.Text());
+        return;
+      }
+    }
+    else if (index == 1 && m_firstChildNode != nullptr) {
+      auto textBlock = this->GetView().as<winrt::TextBlock>();
+      textBlock.ClearValue(winrt::TextBlock::TextProperty());
+      Super::AddView(*m_firstChildNode, 0);
+      m_firstChildNode = nullptr;
```

Comment:
I think you should also null this out in 
```
void ShadowNodeBase::removeAllChildren() {
  GetViewManager()->RemoveAllChildren(GetView());
}
```

override removeAllChildren so you can null it out.